.github/workflows/ci.yml

```diff
@@ -19,6 +19,10 @@ jobs:
     steps:
       - uses: Brightspace/third-party-actions@actions/checkout
       - uses: Brightspace/third-party-actions@actions/setup-dotnet
+        with:
+          dotnet-version: |
+            6.0.x
+            7.0.x

```

Comment:
This codifies the previous expectations of the repo explicitly.
The runner OS images have updated and what versions of .NET are installed has changed, so the implicit assumptions became broken.

We could probably upgrade this package to update something more recent than .NET 6 but realistically there's not much point so for now I'm just maintaining the status quo.